_episodes/06-func.md

```diff
@@ -685,6 +685,25 @@ programmer. If you need to revisit code that you wrote months ago and
 haven't thought about since then, you will appreciate the value of
 readable code!
 
+Python (and most other languages in the C family) provides
+[in-place operators]({{ page.root }}/reference/#in-place-operators)
+that work like this:
```

Comment:
replace `work like this:` with an explanation.
Note, the one given on the reference page is not ideal either. Explain what `+=` does in simple terms.